iiab-install

```diff
@@ -150,7 +150,7 @@ echo -e "Running local playbooks....Stage 0 will now run....followed by Stages $
 
 export ANSIBLE_LOG_PATH="$CWD/iiab-install.log"
 
-ansible -m setup -i $INVENTORY localhost --connection=local >> /dev/null
-ansible-playbook -i $INVENTORY $PLAYBOOK ${ARGS} --connection=local
-
+ansible -m setup -i $INVENTORY localhost --connection=local -e ansible_python_interpreter=/usr/bin/python3 | grep python
+ansible -m setup -i $INVENTORY localhost --connection=local -e ansible_python_interpreter=/usr/bin/python3 >> /dev/null
```

Comment:
```suggestion
```

Reply:
That records the raw ansible varibles to the log file

Reply:
> That records the raw ansible variables to the log file

Should this same line also be in ./iiab-network and ./runrole &mdash; to record var values into their log files too?

Reply:
Given that iiab-network and runrole are a subset of iiab-install I don't feel that is necessary as if we do have an issue it will be during installation.